lib/internal/errors.js

```diff
@@ -838,6 +838,7 @@ E('ERR_FEATURE_UNAVAILABLE_ON_PLATFORM',
   'The feature %s is unavailable on the current platform' +
   ', which is being used to run Node.js',
   TypeError);
+E('ERR_FS_EISDIR', 'is a directory', SystemError);
```

Comment:
I find this error message too terse, can it be expanded at the very least to `Path is a directory` same as it is in the docs? Or there is some other reason for it to be so short?